modules/olm-installing-opm.adoc

```diff
@@ -1,9 +1,9 @@
 // Module included in the following assemblies:
 //
-// * cli_reference/opm-cli.adoc
+// * cli_reference/opm/cli-opm-install.adoc
 
 [id="olm-installing-opm_{context}"]
-= Installing opm
+= Installing the opm CLI
 
 You can install the `opm` CLI tool on your Linux, macOS, or Windows workstation.
```

Comment:
Not technically part of this PR, but I noticed the link in step 1 of the procedure is broken/just not building correctly:  link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-Branch Build/[OpenShift mirror site]

Reply:
Oh yeah, that should be fine on an `enterprise-4.x` branch where `{product-version}` will resolve to a number (instead of "Branch Build" here with a space, which breaks the `link:`).